### ISSUE TITLE
Inquiry about training with orthographic projection camera

### ISSUE THREAD

**Issue by HarryPeverell** [nerfstudio-project/gsplat#582]
Dear gsplat,

I hope this message finds you well. I am writing to seek your assistance regarding an issue I'm encountering while training with an orthographic projection camera. 

I am using the following code snippet for the orthographic camera model during training:

```python
rendered_image, alphas, meta = rasterization(
    means = means3D, # [N, 3]
    quats = rotations, # [N, 4]
    scales = scales, # [N, 3]
    opacities = opacity.squeeze(-1), # [N]
    colors = colors_precomp, # [N, 7]
    viewmats = viewpoint_camera.world_view_transform.transpose(0, 1)[None, ...], # [1, 4, 4]
    Ks = K[None, ...], # [1, 3, 3]
    width = int(viewpoint_camera.image_width),
    height = int(viewpoint_camera.image_height),
    near_plane = viewpoint_camera.znear,
    far_plane = viewpoint_camera.zfar,
    eps2d = 0.3,
    sh_degree = None,
    packed = False,
    backgrounds = bg_color[None, ...],
    camera_model = "ortho" 
)
```
and I would like to clarify that we have not made any modifications to the loss function. And I found that the loss has not shown a significant decrease during training 
```python
        Ll1 = l1_loss(image, gt_image)
        loss = (1.0 - opt.lambda_dssim) * Ll1 + opt.lambda_dssim * (1.0 - ssim(image, gt_image))
        loss.backward()
```
The training results are quite strange, and I suspect it might be due to incorrect orthographic camera parameters. Here are the image parameters I'm using for training:

```json
{
    "camera_projection": "ORTHO",
    "ortho_scale": 5.0,
    "camera_far": 100.0,
    "camera_near": 0.1,
    "frames": [
        {
            "file_ext": ".png",
            "file_path": "./train/r_0",
            "pl_intensity": [
                50000.0,
                50000.0,
                50000.0
            ],
            "transform_matrix": [
                [
                    0.8240397179799347,
                    -0.547227921051675,
                    -0.1466292795215584,
                    -0.7331463976077918
                ],
                [
                    -5.746937261686308e-17,
                    -0.2588190451025207,
                    0.9659258262890682,
                    4.829629131445341
                ],
                [
                    -0.5665320319201286,
                    -0.7959612454847792,
                    -0.21327717293411722,
                    -1.0663858646705857
                ],
                [
                    0.0,
                    0.0,
                    0.0,
                    1.0
                ]
            ]
        }
    ]
}
```
Additionally, I am including the Blender scene setup details for your reference. Please let me know if you need any further information.

![Image](https://github.com/user-attachments/assets/ec5d9415-b50a-4147-95c2-33a3d3ffc0a2)

This is the render image after training:
![Image](https://github.com/user-attachments/assets/e5bbdc9c-b1e7-48ce-ac11-55b8cd9e9680)

Could you please advise if there might be any issues with these camera parameters? Any suggestions on how to improve the training results would be greatly appreciated.

Thank you for your time and expertise.

Best regards,
Harry
